data/map planets.txt

```diff
@@ -277,6 +286,15 @@ planet Baianus
 	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
 	government Uninhabited
 
+planet Bailey
+	attributes ccor mining
+	landscape land/lava3
+	description `Bailey is a dead, nearly tidally locked world scarcely larger and more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Despite its inhospitable conditions, Bailey's bountiful crust has brought several mining expeditions to the planet.`
+	description `	Like aphids sucking the sap from a plant, semi-mobile mining stations strip and refine the minerals, a method originally developed by the Confederated Councils. Since their fall, the pirates of the region have maintained the stations - although with considerably less organization than what was upheld by the central government.`
+	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located on a repulsor-lift facility hovering a meter above Bailey's surface. Floating in place thanks to the lack of atmosphere, the planet slowly rotates beneath it while the facility is always on Bailey's dark side.`
```

Comment:
There's something I'm still not understanding here. If the planet is nearly tidally locked, then how does the spaceport remain on the dark side? In tidal locking, the planet rotates about its own axis with a rotational period equal to its orbital period around its star. If the planet is allowed to rotate beneath the station, then wouldn't the station eventually reach the hot side of the planet?

Let's say the planet looks superficially like Earth, and the station is initially hovering 1 meter above Sri Lanka on the night side at time T-0. By time T-12, the tidally locked Earth-like planet has both rotated and orbited its star, such that Sri Lanka is still cloaked in darkness. However, the station has remained stagnant and is now covered in sunlight.

What am I missing?

Reply:
I think the facility moves like the settlements do?

Reply:
Floating in place means the planet rotates, while the facility being repulsorlift off of the ground, does not rotate to it as it's not attached.

Reply:
> so it does not rotate with the planet as it's not attached to the surface but hovering just above it
That's kind of the problem (as I see it). If it does not rotate, then it will eventually end up in the sunlight. It _has_ to rotate somewhat to remain in the darkness, because the planet rotates in order to remain in the darkness.

Still not sure if I fully understand, but I'm ok with hand-waving this if it's not a big deal.

Reply:
> > so it does not rotate with the planet as it's not attached to the surface but hovering just above it
> 
> That's kind of the problem (as I see it). If it does not rotate, then it will eventually end up in the sunlight. It _has_ to rotate somewhat to remain in the darkness, because the planet rotates in order to remain in the darkness.
> 
> Still not sure if I fully understand, but I'm ok with hand-waving this if it's not a big deal.

The problem here is that Bailey is _almost_ tidally locked but not entirely. It'd be something like a day taking 0.99 or 1.01 local years. We're in something approaching a damned if you do/damned if you don't state with the Spaceport.
In technicality it'd be rotating at 1 to 1 with the orbital period in spite of the planet's differing rate... not sure how to say that in a concise non-clunky manner.

Reply:
Ah, okay. Then I withdraw my original comment. Thanks for clarifying!